etcdutl/snapshot/v3_snapshot.go

```diff
@@ -203,6 +204,16 @@ type RestoreConfig struct {
 	// SkipHashCheck is "true" to ignore snapshot integrity hash value
 	// (required if copied from data directory).
 	SkipHashCheck bool
+
+	// RevisionBump is the amount to increase the latest revision after restore,
+	// to ensure that the revisions never decrease from the clients perspective
+	// If 0, revision bumping is skipped.
+	RevisionBump int64
+
+	// MarkCompacted is "true" to mark the latest revision as scheduled for compaction.
+	// This will cause the server to compact the restored revisions (at some point after startup),
+	// and correctly cancel watchers so they can invalidate their caches.
```

Comment:
Comments are nice, but it would be better to just provide all this information as part of user visible information in flag help section.

Reply:
as commented earlier, I think this is too focused on K8s as a consumer. In both places, I would leave that information out.